src/applications/appeals/995/tests/utils/submit.unit.spec.js

```diff
@@ -338,6 +366,18 @@ describe('getEvidence', () => {
               ],
             },
           },
+          {
+            type: 'retrievalEvidence',
+            attributes: {
+              locationAndName: 'test 3',
+              evidenceDates: [
+                {
+                  startDate: newForm ? TEMP_DATE : '2022-05-05',
+                  endDate: newForm ? TEMP_DATE : '2022-06-06',
```

Comment:
Will it be a problem on the backend if we use the same TEMP_DATE for start and end?

Reply:
It shouldn't be a problem. I've had this method in place so we can submit new data to the current API